forge/build.gradle

```
@@ -22,7 +22,9 @@ loom {
 }
 
 dependencies {
-    forge("net.minecraftforge:forge:${rootProject.minecraft_version}-${rootProject.forge_version}")
+    forge "net.minecraftforge:forge:${libs.versions.minecraft.get()}-${libs.versions.forge.get()}"
+
+//    modRuntimeOnly project(":TestForgeMod")
```

Rogue comment

---

Actually I see what you did here and that is cool. Add a comment then. 
```suggestion
// Uncomment this to make the test mod run alongside NEC. 
//    modRuntimeOnly project(":TestForgeMod")
```